Documentation/guides/DotNet5.md

```diff
@@ -45,6 +71,14 @@ In .NET 5 the behavior of the following MSBuild tasks will change, but
 better align with existing .NET 5 semantics, at the cost of compatibility with
 previous Xamarin.Android releases.
 
+`$(AndroidClassParser)` will be `class-parse` by default.
```

Comment:
This should mention that not only is `class-parse` the default, but that `jar2xml` is not supported at all.

Reply:
Same for `$(AndroidCodegenTarget)`.